packages/salesforcedx-sobjects-faux-generator/test/integration/fauxGenerate.test.ts

```diff
@@ -99,7 +99,6 @@ describe('Generate faux classes for SObjects', function() {
   });
 
   it('Should emit an error event on failure', async () => {
-    let result = '';
```

Comment:
The new typescript + tslint analyzer is not very aggressive in pruning unread variables.

If you only write to it but never read from it, it's considered an unused variable.